package/com.unity.formats.usd/Documentation~/USD.md

```diff
@@ -1,126 +1,114 @@
-the inspector on the game object at which these features were authored. Note that Payloads *are not loaded by default*. USD files using Payloads must be reloaded after changing the Payload Policy in the Inspector to 'Load All' for them to appear in the Scene. 
-
-## Exporting USD files
-
-USD files can be created by exporting GameObjects selected in the Hierarchy using the USD context menu options, or using the Recorder.
-
-### Exporting via Recorder
-
-> **Prerequisite:** You need to install the [Recorder package](https://docs.unity3d.com/Packages/com.unity.recorder@latest/index.html).  
-
-To export a USD composition via Recorder, you can either use the Recorder Window :
-* From the Editor main menu, select **Window > General > Recorder > Recorder Window**,
-* Then click on **+ Add Recorder**,
-* And select **USD Clip**.
-
-Or add a Recorder track:
-* From the Timeline window, right-click and select **UnityEditor.Recorder.Timeline > Recorder Track**,
-* Then right-click on the track and **add Recorder Clip**,
-* And in the **selected recorder**, choose **USD Clip**.
-
-Using Recorder is the recommended option *when compatibility with runtime is not required*.
-
-### Exporting via Usd Recorder Track (Legacy)
-
-When compatibility with runtime is required (i.e for a standalone build), the recommended option is to use the USD package Recorder Track :
-* From the Timeline window, right-click and select **Unity.Formats.USD > Usd Recorder Track**,
-* Then right-click on the track and **add USD Recorder Clip**.
-
->  **Note:** This feature has no dependency to and is not based on the Recorder package.
-
+* Importing and exporting USD:
+    * Import as GameObject, Prefab, or Timeline Clip
+        * Formats: USDA, USDC, USDZ
+    * Export Game Objects to USD
+        * Formats: USDA, USDC, USDZ
+        * Export of Transform Overrides
+    * Export via Recorder package
+* Composition:
+    * Variant Selection
+    * Payloads
+        * Load All and Individual
+    * Layer stacks
+* Geometry:
+    * UV Set
+    * Vertex Colour
+* Instancing:
+    * Point Instancing
+    * Scene Instancing
+* Primitive Types:
+    * Meshes:
+        * Arbitrary Primvars
+        * Vertex Colors
+    * Materials:
+        * Standard Shader and Limited HDRP and URP Support
+* Cameras
+* Lightmaps:
+    * Automatic Lightmap UV Unwrapping
+* Animation:
+    * Timeline Playback:
+        * Skeletal Animation via USDSkel
+        * Animated Meshes
+    * Timeline Recording Track via Unity Recorder Package
+* General:
+    * High and Low Level Access to USD API via C#
+
+## Known Limitations
+
+We do not currently support the following:
+
+* Geometry:
+    * Multiple UV Sets
+    * Importing mesh UVs for a prim without a material attached
+* Composition:
+    * Purposes
+* Primitive Types:
+    * Camera
+        * We do not currently import and export all physical camera settings
+    * Materials:
+        * Transparency Settings (these can be set manually)
```

Comment:
Transparency & Double-Sided

Reply:
Just in case there's other things we're missing that we don't even know about, I'll keep this vague and put them as examples?